common/ModRbc.F90

```diff
@@ -175,6 +177,131 @@ subroutine RBC_MakeSphere(cell, r, xc)
     end if
 
   end subroutine RBC_MakeSphere
+  
+  ! Subroutine creatse an uncurved shape with the size of a sickle cell. This prolate spheroid shape is then
+  ! run through a flow to create a curve in the shape. The resulting curved shape after the flow was then 
+  ! exported and used as a sickle cell.
+  ! Reference:
+  !   Xiao Zhang, Wilbur A. Lam and Michael D. Graham
+  !   Dynamics of deformable straight and curved prolate capsules in simple shear flow
+  !   PHYSICAL REVIEW FLUIDS 5, 053101 (2020)
+  subroutine RBC_MakeUncurvedSickleSpheroid(cell, r, xc)
+  
+    type(t_RBC) :: cell
+    real(WP) :: r
+    real(WP),optional :: xc(3)
+    integer :: ilat, ilon, ii
+    real(WP) :: th, phi
+    real(WP) :: pol, eq, curv
+  
+    pol = 1.2 * r
+    eq = 0.25 * r
+    curv = 0.2 * r
+  
+    do ilon = 1, cell%nlon
+    do ilat = 1, cell%nlat
+      th = cell%th(ilat)
+      phi = cell%phi(ilon)
+      cell%x(ilat,ilon,1) = pol*sin(th)*cos(phi)
+      cell%x(ilat,ilon,2) = eq*sin(th)*sin(phi)
+      cell%x(ilat,ilon,3) = eq*cos(th)
+  
+      !curve capsule
+      !cell%x(ilon, ilat, 2) = cell%x(ilon, ilat, 2) - (curv * (cell%x(ilon, ilat, 3)**2))
+    end do ! ilat
+    end do ! ilon
+    if (present(xc)) then
+      do ii = 1, 3
+        cell%x(:,:,ii) = cell%x(:,:,ii) + xc(ii)
+      end do ! ii
+    end if
+  
+    end subroutine RBC_MakeUncurvedSickleSpheroid    
+
+  ! Make Sickle Cell based on the shape given by Reference:
+  !   Kviatkovsky, I. Zeidan, A. Yeheskely-Hayon, D. Shabad, E. L. Dann, E. J. & Yelin, D. 
+  !   Measuring Sickle Cell morphology during blood flow
+  !   Biomedical optics express, 8(3), 1996–2003
+  !   doi: 10.1364/BOE.8.001996
+  ! Unfortunately this shape doesn't work very well since points are ill-conditioned
+  ! and diverge for this type of simulation, so we do not proceed with this sickle-cell model
+
+  ! subroutine RBC_MakeSickle(cell, rad, xc)
```

Comment:
uncomment this and then add an `write('this configuration is not confirmed working yet'` and an `mpi_abort()` call. do not leave stale code in the codebase.

Reply:
Will do this in rev3